src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/EventPublisherContainer.java

```diff
@@ -44,8 +44,8 @@ public EventPublisher getPublisher(
                     caseDetails
                 );
             case EXCEPTION:
-                return exceptionRecordCreator;
             case NEW_APPLICATION:
+                return exceptionRecordCreator;
```

Comment:
wait a minute. new application case type is same/similar as exception record?

Reply:
Yes. Felt weird to me, too. Data will be different, but the orchestrator is supposed to process those exactly the same way.

Reply:
only difference is exception may be for supplementary evidence or it may be for new application. So when it is exception classification orchestrator should send everything.